ccl/changefeedccl: TestAvroWithRegionalTable failed

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15000876?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15000876?buildTab=artifacts#/) on master @ [69e1fce38afbbb02b5d1de48355f501604144a8a](https://github.com/cockroachdb/cockroach/commits/69e1fce38afbbb02b5d1de48355f501604144a8a):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 130917 [chan receive]:
        testing.(*T).Parallel(0x4006c1b380)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4006c1b380)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4006c1b380, 0x6475610)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 130919 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4e8c0, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130921 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4ee00, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130920 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4ed20, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130918 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4e380, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 130918 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4e380, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130919 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4e8c0, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130920 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4ed20, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130921 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400ac4ee00, {0x78eced8, 0x4009e05040})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130917
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (29.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-38214

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15003106?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15003106?buildTab=artifacts#/) on master @ [507b66b75ee478a0007a96b40b831b72eb3c1b4d](https://github.com/cockroachdb/cockroach/commits/507b66b75ee478a0007a96b40b831b72eb3c1b4d):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 126195 [chan receive]:
        testing.(*T).Parallel(0x400e9b6000)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400e9b6000)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400e9b6000, 0x6475610)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 126199 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b9822a0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 126197 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b982000, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 126196 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a62fea0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 126198 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b9821c0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 126196 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a62fea0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 126197 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b982000, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 126198 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b9821c0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 126199 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400b9822a0, {0x78eced8, 0x40096cae10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 126195
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (56.67s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15003550?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15003550?buildTab=artifacts#/) on master @ [25dff3bcacb3d447d4ce4cb0cefb945768249102](https://github.com/cockroachdb/cockroach/commits/25dff3bcacb3d447d4ce4cb0cefb945768249102):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 146498 [chan receive]:
        testing.(*T).Parallel(0x400b498b60)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400b498b60)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400b498b60, 0x6476250)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 146499 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a460, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 146501 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a8c0, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 146502 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798aa80, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 146500 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a700, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 146499 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a460, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 146500 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a700, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 146501 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798a8c0, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 146502 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400798aa80, {0x78edd38, 0x4009721a90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 146498
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (42.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15006298?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15006298?buildTab=artifacts#/) on master @ [6f3c52d09f175e33ca2b4bced547fbfe55e4a39c](https://github.com/cockroachdb/cockroach/commits/6f3c52d09f175e33ca2b4bced547fbfe55e4a39c):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 121891 [chan receive]:
        testing.(*T).Parallel(0x400c5c6ea0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400c5c6ea0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400c5c6ea0, 0x6476250)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 121893 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6460, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 121892 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc62a0, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 121895 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6700, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 121894 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6620, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 121892 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc62a0, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 121893 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6460, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 121894 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6620, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 121895 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400abc6700, {0x78edd38, 0x400d0c2b90})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 121891
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (47.86s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15007521?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15007521?buildTab=artifacts#/) on master @ [5d02bd9ff6b2bccecf6d43fc6cd647167b91f782](https://github.com/cockroachdb/cockroach/commits/5d02bd9ff6b2bccecf6d43fc6cd647167b91f782):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 148576 [chan receive]:
        testing.(*T).Parallel(0x400b77eb60)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400b77eb60)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400b77eb60, 0x6476250)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 148723 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e242a0, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 148724 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e24380, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 148722 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e241c0, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 148577 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e24000, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 148577 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e24000, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 148722 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e241c0, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 148723 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e242a0, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 148724 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012e24380, {0x78edd38, 0x400bf65e50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 148576
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (54.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://tanzanite.cluster.engflow.com/invocations/default/6ef85470-d91a-493d-8172-1e90200d6fc6?testReportRun=12&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [5d02bd9ff6b2bccecf6d43fc6cd647167b91f782](https://github.com/cockroachdb/cockroach/commits/5d02bd9ff6b2bccecf6d43fc6cd647167b91f782):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 115662 [chan receive]:
        testing.(*T).Parallel(0xc0084a64e0)
        	GOROOT/src/testing/testing.go:1483 +0x215
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0xc0084a64e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xc8
        testing.tRunner(0xc0084a64e0, 0x6c36068)
        	GOROOT/src/testing/testing.go:1689 +0xfb
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x390
        goroutine 115665 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e95e0, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 115663 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e9420, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 115682 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e96c0, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 115664 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e9500, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f: Leaked goroutine: goroutine 115663 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e9420, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 115664 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e9500, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 115665 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e95e0, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 115682 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00b3e96c0, {0x80bda20, 0xc0098e8780})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115662
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
--- FAIL: TestAvroWithRegionalTable (32.93s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=12</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://tanzanite.cluster.engflow.com/invocations/default/1961cc0d-6303-4f8e-b6fe-c19dc9f7950a?testReportRun=3&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [5d02bd9ff6b2bccecf6d43fc6cd647167b91f782](https://github.com/cockroachdb/cockroach/commits/5d02bd9ff6b2bccecf6d43fc6cd647167b91f782):


```
=== RUN   TestAvroWithRegionalTable
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestAvroWithRegionalTable1011426558
    test_log_scope.go:81: use -show-logs to present logs inline
    encoder_test.go:1313: -- test log scope end --
test logs left over in: outputs.zip/logTestAvroWithRegionalTable1011426558
--- FAIL: TestAvroWithRegionalTable (164.28s)
=== RUN   TestAvroWithRegionalTable/key_only
    --- FAIL: TestAvroWithRegionalTable/key_only (55.81s)
=== RUN   TestAvroWithRegionalTable/key_only/overrideWithSingleWorker=true
    encoder_test.go:1291: overriding number of parallel workers to one
    encoder_test.go:1309: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:199
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:274
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_test.go:1309
        	            				github.com/cockroachdb/cockroach/pkg/testutils/subtest.go:28
        	Error:      	Received unexpected error:
        	            	cannot unmarshal schema JSON: unexpected end of JSON input
        	Test:       	TestAvroWithRegionalTable/key_only/overrideWithSingleWorker=true
        --- FAIL: TestAvroWithRegionalTable/key_only/overrideWithSingleWorker=true (28.19s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://tanzanite.cluster.engflow.com/invocations/default/3a916357-362a-4e40-aed8-d56fa3de00e8?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [2b5baff23f9b53da26f0fc1a763ca43384b464b0](https://github.com/cockroachdb/cockroach/commits/2b5baff23f9b53da26f0fc1a763ca43384b464b0):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 89931 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41500, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 89928 [chan receive]:
        testing.(*T).Parallel(0xc00bdc24e0)
        	GOROOT/src/testing/testing.go:1483 +0x215
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0xc00bdc24e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xc8
        testing.tRunner(0xc00bdc24e0, 0x6c36068)
        	GOROOT/src/testing/testing.go:1689 +0xfb
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x390
        goroutine 89930 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41420, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 89932 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f416c0, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 89929 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41340, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f: Leaked goroutine: goroutine 89929 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41340, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 89930 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41420, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 89931 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f41500, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 89932 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc008f416c0, {0x80bda20, 0xc009019720})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 89928
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
--- FAIL: TestAvroWithRegionalTable (38.32s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15028551?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15028551?buildTab=artifacts#/) on master @ [641407f699a9c3ab03dbd70f337dc869cb490b41](https://github.com/cockroachdb/cockroach/commits/641407f699a9c3ab03dbd70f337dc869cb490b41):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 101468 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c781c0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 101470 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c78380, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 101467 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c780e0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 101469 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c782a0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 101466 [chan receive]:
        testing.(*T).Parallel(0x4003f3c340)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4003f3c340)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4003f3c340, 0x6476250)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318: Leaked goroutine: goroutine 101467 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c780e0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 101468 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c781c0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 101469 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c782a0, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 101470 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007c78380, {0x78edd38, 0x40074ff900})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 101466
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (43.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15030553?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15030553?buildTab=artifacts#/) on master @ [39755f13dc663f54b0d8a7f19bd645072112fee1](https://github.com/cockroachdb/cockroach/commits/39755f13dc663f54b0d8a7f19bd645072112fee1):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 108842 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7180, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108845 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7420, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108841 [chan receive]:
        testing.(*T).Parallel(0x4005c16680)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4005c16680)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4005c16680, 0x64762b0)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 108844 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7340, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108843 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7260, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 108842 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7180, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108843 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7260, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108844 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7340, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108845 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40073c7420, {0x78edeb8, 0x40090ed770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108841
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (40.14s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15031696?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15031696?buildTab=artifacts#/) on master @ [978a6a5d045c36401468b4010bbbb35cbadf2093](https://github.com/cockroachdb/cockroach/commits/978a6a5d045c36401468b4010bbbb35cbadf2093):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 108530 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259340, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108449 [chan receive, 1 minutes]:
        testing.(*T).Parallel(0x400ecc3520)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400ecc3520)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400ecc3520, 0x64764f0)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 108532 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259500, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108531 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259420, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108533 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f2595e0, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 108530 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259340, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108531 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259420, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108532 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f259500, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108533 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400f2595e0, {0x78ee0d8, 0x400bfbd540})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108449
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (73.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15032215?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15032215?buildTab=artifacts#/) on master @ [a54f0e0013185abbf3b4c71d19756af072e2d37f](https://github.com/cockroachdb/cockroach/commits/a54f0e0013185abbf3b4c71d19756af072e2d37f):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 130641 [chan receive]:
        testing.(*T).Parallel(0x4006287520)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4006287520)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4006287520, 0x64764f0)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 130834 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009162fc0, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130835 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40091630a0, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130837 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009163260, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 130836 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009163180, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 130834 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009162fc0, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130835 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40091630a0, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130836 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009163180, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 130837 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009163260, {0x78ee0d8, 0x4006e3a4b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 130641
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (40.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15033421?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15033421?buildTab=artifacts#/) on master @ [20749b8638e151dcf464b2795d05e28d5645489a](https://github.com/cockroachdb/cockroach/commits/20749b8638e151dcf464b2795d05e28d5645489a):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 157052 [chan receive]:
        testing.(*T).Parallel(0x400a26ed00)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400a26ed00)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400a26ed00, 0x6476590)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 157053 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006f6bdc0, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 157055 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40014a2380, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 157054 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006f6bea0, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 157056 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40014a2460, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 157053 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006f6bdc0, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 157054 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006f6bea0, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 157055 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40014a2380, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 157056 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40014a2460, {0x78ee198, 0x400833e5a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 157052
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (37.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15034342?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15034342?buildTab=artifacts#/) on master @ [7cd5b546000b5999c6acb500f3db337985a8abeb](https://github.com/cockroachdb/cockroach/commits/7cd5b546000b5999c6acb500f3db337985a8abeb):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 122955 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3a40, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122957 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3c00, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122958 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3ce0, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122954 [chan receive]:
        testing.(*T).Parallel(0x4008e5e820)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4008e5e820)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4008e5e820, 0x6476198)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 122956 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3b20, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 122955 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3a40, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122956 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3b20, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122957 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3c00, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122958 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c2f3ce0, {0x78eda78, 0x400d2838b0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122954
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (52.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15035671?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15035671?buildTab=artifacts#/) on master @ [a730c5689f39b604e1d19de232359f94339fec0f](https://github.com/cockroachdb/cockroach/commits/a730c5689f39b604e1d19de232359f94339fec0f):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 102669 [chan receive]:
        testing.(*T).Parallel(0x4006a42000)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4006a42000)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4006a42000, 0x64765b8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 102672 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a212380, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102670 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a2121c0, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102673 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a212460, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102671 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a2122a0, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 102670 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a2121c0, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102671 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a2122a0, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102672 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a212380, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102673 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400a212460, {0x78ee378, 0x400bc0e050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102669
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (50.62s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15040055?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15040055?buildTab=artifacts#/) on master @ [5a4bae3920f918ad2b31653aa65d4648c6020fe5](https://github.com/cockroachdb/cockroach/commits/5a4bae3920f918ad2b31653aa65d4648c6020fe5):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 113490 [chan receive, 1 minutes]:
        testing.(*T).Parallel(0x4005719380)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4005719380)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4005719380, 0x6476c18)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 113491 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f4fc0, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 113492 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f50a0, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 113493 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f5180, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 113494 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f5340, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 113491 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f4fc0, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 113492 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f50a0, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 113493 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f5180, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 113494 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40086f5340, {0x78eea38, 0x400c7b1bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 113490
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (72.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15040697?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15040697?buildTab=artifacts#/) on master @ [2a6bd6f3a14d7ddb98e293bf01b4f9c1f9edf6ff](https://github.com/cockroachdb/cockroach/commits/2a6bd6f3a14d7ddb98e293bf01b4f9c1f9edf6ff):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 107130 [chan receive]:
        testing.(*T).Parallel(0x400afcda00)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400afcda00)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400afcda00, 0x6476c18)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 107132 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcad20, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 107131 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcac40, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 107133 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcae00, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 107134 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcaee0, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 107131 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcac40, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 107132 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcad20, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 107133 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcae00, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 107134 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009dcaee0, {0x78eea38, 0x40099b66e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 107130
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (29.50s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15041443?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15041443?buildTab=artifacts#/) on master @ [145ac765c8a829cf167d79f5842133ce6bf35e57](https://github.com/cockroachdb/cockroach/commits/145ac765c8a829cf167d79f5842133ce6bf35e57):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 122747 [chan receive]:
        testing.(*T).Parallel(0x400ec34680)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400ec34680)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400ec34680, 0x6476df8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 122748 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f6e00, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122751 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f7180, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122750 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f70a0, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122749 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f6ee0, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 122748 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f6e00, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122749 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f6ee0, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122750 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f70a0, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122751 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093f7180, {0x78eec78, 0x400a1d2a50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122747
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (46.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15041923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15041923?buildTab=artifacts#/) on master @ [be5469253232fed0c53d01b4b7afc58c62efba1e](https://github.com/cockroachdb/cockroach/commits/be5469253232fed0c53d01b4b7afc58c62efba1e):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 102322 [chan receive]:
        testing.(*T).Parallel(0x400c7be000)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400c7be000)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400c7be000, 0x6476e38)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 102324 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41420, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102326 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a415e0, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102323 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41340, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 102325 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41500, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 102323 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41340, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102324 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41420, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102325 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a41500, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 102326 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009a415e0, {0x78eecd8, 0x400ad34f00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 102322
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (51.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://tanzanite.cluster.engflow.com/invocations/default/62643dc2-99b3-4250-b096-2d2f40731773?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [be5469253232fed0c53d01b4b7afc58c62efba1e](https://github.com/cockroachdb/cockroach/commits/be5469253232fed0c53d01b4b7afc58c62efba1e):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 108613 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40b60, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 108615 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40d20, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 108614 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40c40, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        goroutine 108612 [chan receive]:
        testing.(*T).Parallel(0xc00be70820)
        	GOROOT/src/testing/testing.go:1483 +0x215
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0xc00be70820)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xc8
        testing.tRunner(0xc00be70820, 0x6c36aa8)
        	GOROOT/src/testing/testing.go:1689 +0xfb
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x390
        goroutine 108616 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40e00, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f: Leaked goroutine: goroutine 108613 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40b60, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 108614 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40c40, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 108615 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40d20, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
        Leaked goroutine: goroutine 108616 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0xc00bc40e00, {0x80be800, 0xc009d46050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x79
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108612
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0x121f
--- FAIL: TestAvroWithRegionalTable (38.50s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15043550?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15043550?buildTab=artifacts#/) on master @ [086f930e2b94913737db9ef9877d10976355874c](https://github.com/cockroachdb/cockroach/commits/086f930e2b94913737db9ef9877d10976355874c):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 125657 [chan receive]:
        testing.(*T).Parallel(0x400a986680)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400a986680)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400a986680, 0x64771f8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 125658 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e094620, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125660 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e0948c0, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125659 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e094700, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125661 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e0949a0, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 125658 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e094620, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125659 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e094700, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125660 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e0948c0, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125661 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400e0949a0, {0x78eefb8, 0x4010faef00})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125657
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (46.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15044582?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15044582?buildTab=artifacts#/) on master @ [32e46c206ae11a038caf2a6773b1c3db5920421d](https://github.com/cockroachdb/cockroach/commits/32e46c206ae11a038caf2a6773b1c3db5920421d):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 90929 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2db20, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 90926 [chan receive]:
        testing.(*T).Parallel(0x4005baf1e0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4005baf1e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x4005baf1e0, 0x6476ef8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 90928 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2da40, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 90927 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2d960, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 91442 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2dc00, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 90927 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2d960, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 90928 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2da40, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 90929 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2db20, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 91442 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400cc2dc00, {0x78eebb8, 0x40104dd090})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 90926
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (43.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15049200?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15049200?buildTab=artifacts#/) on master @ [16b8102a12a00b0e5e1c3540158067e49ccfb5f0](https://github.com/cockroachdb/cockroach/commits/16b8102a12a00b0e5e1c3540158067e49ccfb5f0):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 119677 [chan receive]:
        testing.(*T).Parallel(0x400ae151e0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400ae151e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:34 +0xa4
        testing.tRunner(0x400ae151e0, 0x64763f8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 119680 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40074675e0, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 119679 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467500, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 119681 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467880, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 119678 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467340, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 119678 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467340, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 119679 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467500, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 119680 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40074675e0, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 119681 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007467880, {0x78ed6d8, 0x4004524d70})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 119677
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (47.25s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15050752?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15050752?buildTab=artifacts#/) on master @ [c1ce9879c3f9f0b5fc1257a262799659859aad5e](https://github.com/cockroachdb/cockroach/commits/c1ce9879c3f9f0b5fc1257a262799659859aad5e):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 108621 [chan receive]:
        testing.(*T).Parallel(0x40071181a0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x40071181a0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x40071181a0, 0x64764f0)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 108622 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba0e0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108624 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba2a0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108625 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba380, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 108623 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba1c0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 108622 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba0e0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108623 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba1c0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108624 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba2a0, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 108625 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4009cba380, {0x78ed798, 0x4009ef2050})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 108621
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (30.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15051776?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15051776?buildTab=artifacts#/) on master @ [38b8c5e2f29bca0575d6118d3a9edb6783c89ca5](https://github.com/cockroachdb/cockroach/commits/38b8c5e2f29bca0575d6118d3a9edb6783c89ca5):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 98658 [chan receive, 1 minutes]:
        testing.(*T).Parallel(0x400a329ba0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400a329ba0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x400a329ba0, 0x6478958)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 98661 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b3880, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 98659 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b36c0, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 98662 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b3960, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 98660 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b37a0, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 98659 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b36c0, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 98660 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b37a0, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 98661 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b3880, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 98662 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40036b3960, {0x78efe18, 0x400c81fd10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 98658
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (61.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15052473?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15052473?buildTab=artifacts#/) on master @ [4a7029631bbec84e06e6325655019ac83d878120](https://github.com/cockroachdb/cockroach/commits/4a7029631bbec84e06e6325655019ac83d878120):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 122226 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc0e0, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122127 [chan receive]:
        testing.(*T).Parallel(0x40082f1520)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x40082f1520)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x40082f1520, 0x6478ef8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 122227 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc1c0, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122129 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc000, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 122128 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40015a4b60, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 122128 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40015a4b60, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122129 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc000, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122226 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc0e0, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 122227 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4007ffc1c0, {0x78f03f8, 0x4009caaa50})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 122127
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (46.29s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15053325?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15053325?buildTab=artifacts#/) on master @ [4a7029631bbec84e06e6325655019ac83d878120](https://github.com/cockroachdb/cockroach/commits/4a7029631bbec84e06e6325655019ac83d878120):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 109605 [chan receive]:
        testing.(*T).Parallel(0x400985e820)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x400985e820)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x400985e820, 0x6478ef8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 109606 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40125902a0, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 109608 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590460, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 109607 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590380, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 109609 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590700, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 109606 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40125902a0, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 109607 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590380, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 109608 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590460, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 109609 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4012590700, {0x78f03f8, 0x4001668cd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 109605
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (31.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15056128?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15056128?buildTab=artifacts#/) on master @ [82bbf6b1021b84cbfa5ad60f95e4a996372cb422](https://github.com/cockroachdb/cockroach/commits/82bbf6b1021b84cbfa5ad60f95e4a996372cb422):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 105643 [chan receive, 1 minutes]:
        testing.(*T).Parallel(0x4003e9d380)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4003e9d380)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x4003e9d380, 0x647c2d8)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 105644 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525b20, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 105647 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525ea0, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 105646 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525ce0, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 105645 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525c00, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 105644 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525b20, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 105645 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525c00, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 105646 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525ce0, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 105647 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c525ea0, {0x78f3ff8, 0x4008e8f770})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 105643
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (74.90s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15056963?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15056963?buildTab=artifacts#/) on master @ [e57234922d9367a92a94c3b324a545ea3aac0ba5](https://github.com/cockroachdb/cockroach/commits/e57234922d9367a92a94c3b324a545ea3aac0ba5):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 110507 [chan receive]:
        testing.(*T).Parallel(0x40092d64e0)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x40092d64e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x40092d64e0, 0x647c358)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 110510 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81dc0, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 110511 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81ea0, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 110508 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81b20, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 110509 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81c00, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 110508 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81b20, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 110509 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81c00, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 110510 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81dc0, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 110511 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4005c81ea0, {0x78f3ef8, 0x400c503d10})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 110507
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (33.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15057280?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15057280?buildTab=artifacts#/) on master @ [186b9b9c9f560d6966d1237993713ff2d26ae2dc](https://github.com/cockroachdb/cockroach/commits/186b9b9c9f560d6966d1237993713ff2d26ae2dc):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 125637 [chan receive]:
        testing.(*T).Parallel(0x4007129a00)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4007129a00)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x4007129a00, 0x647d218)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 125639 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e460, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125641 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e620, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125640 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e540, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125638 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e380, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 125638 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e380, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125639 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e460, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125640 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e540, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125641 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x400c99e620, {0x78f4f58, 0x400ad79a40})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125637
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (32.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15063509?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15063509?buildTab=artifacts#/) on master @ [5c8a9fe2ded3e0e2eca5cbae258c57c214242bca](https://github.com/cockroachdb/cockroach/commits/5c8a9fe2ded3e0e2eca5cbae258c57c214242bca):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 115924 [chan receive, 1 minutes]:
        testing.(*T).Parallel(0x401301c340)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x401301c340)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x401301c340, 0x647d358)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 115926 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3960, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115925 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3880, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115928 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3b20, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115927 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3a40, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 115925 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3880, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115926 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3960, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115927 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3a40, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115928 [select, 1 minutes]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40093b3b20, {0x78f50d8, 0x40017f90e0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115924
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (67.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15065557?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15065557?buildTab=artifacts#/) on master @ [db0b5e112a20aeb5109013dfbdd8682e2b36fe8b](https://github.com/cockroachdb/cockroach/commits/db0b5e112a20aeb5109013dfbdd8682e2b36fe8b):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 115898 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4008856540, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115897 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4008856460, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115895 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40088561c0, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 115894 [chan receive]:
        testing.(*T).Parallel(0x4009a7e000)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x4009a7e000)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x4009a7e000, 0x647dd18)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 115896 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40088562a0, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 115895 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40088561c0, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115896 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x40088562a0, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115897 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4008856460, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 115898 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4008856540, {0x78f5b98, 0x400af660a0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 115894
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (40.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestAvroWithRegionalTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15066969?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15066969?buildTab=artifacts#/) on master @ [683589658929ced6ca78075ab40ca76a4bc6182c](https://github.com/cockroachdb/cockroach/commits/683589658929ced6ca78075ab40ca76a4bc6182c):


```
        	github.com/bazelbuild/rules_go/go/tools/bzltestutil/external/io_bazel_rules_go/go/tools/bzltestutil/timeout.go:33 +0x98
        goroutine 125301 [chan receive]:
        testing.(*T).Parallel(0x40062cb860)
        	GOROOT/src/testing/testing.go:1483 +0x1ac
        github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestJSONEncoderJSONNullAsObject(0x40062cb860)
        	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/encoder_json_test.go:35 +0xa4
        testing.tRunner(0x40062cb860, 0x6480a78)
        	GOROOT/src/testing/testing.go:1689 +0xec
        created by testing.(*T).Run in goroutine 1
        	GOROOT/src/testing/testing.go:1742 +0x318
        goroutine 125303 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d507e0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125304 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d508c0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125305 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d509a0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        goroutine 125302 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d50700, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac: Leaked goroutine: goroutine 125302 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d50700, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125303 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d507e0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125304 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d508c0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
        Leaked goroutine: goroutine 125305 [select]:
        github.com/cockroachdb/cockroach/pkg/util/log.(*fileSink).gcDaemon(0x4006d509a0, {0x78f8bb8, 0x40017a9bd0})
        	github.com/cockroachdb/cockroach/pkg/util/log/file_log_gc.go:25 +0x70
        created by github.com/cockroachdb/cockroach/pkg/util/log.ApplyConfig in goroutine 125301
        	github.com/cockroachdb/cockroach/pkg/util/log/flags.go:328 +0xeac
--- FAIL: TestAvroWithRegionalTable (38.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAvroWithRegionalTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/ccl/changefeedccl/encoder_json_test.go
+++ b/pkg/ccl/changefeedccl/encoder_json_test.go
@@ -19,7 +19,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog"
 	"github.com/cockroachdb/cockroach/pkg/sql/rowenc"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/json"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
@@ -31,8 +30,6 @@ import (
 func TestJSONEncoderJSONNullAsObject(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-
-	t.Parallel() // SAFE FOR TESTING
 
 	ctx := context.Background()
 	tableDesc, err := parseTableDesc(`CREATE TABLE foo (a JSONB PRIMARY KEY, b JSONB)`)
@@ -177,10 +174,6 @@ func TestJSONEncoderJSONNullAsObject(t *testing.T) {
 func TestJSONEncoderJSONNullAsObjectEdgeCases(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-
-	skip.WithIssue(t, 123156)
-
-	t.Parallel() // SAFE FOR TESTING
 
 	ctx := context.Background()
 	ts := hlc.Timestamp{WallTime: 1, Logical: 2}

